apps/worker/src/config/env.validators.ts

```diff
@@ -54,6 +54,7 @@ export const envValidators = {
   MONGO_URL: str(),
   MONGO_MIN_POOL_SIZE: num({ default: 10 }),
   MONGO_MAX_POOL_SIZE: num({ default: 500 }),
+  MONGO_MAX_IDLE_TIME_IN_MS: num({ default: 1000 * 30 }),
```

Comment:
Adding the missing env vars for auto index creation

```suggestion
  MONGO_MAX_IDLE_TIME_IN_MS: num({ default: 1000 * 30 }),
  /** @deprecated - use `MONGO_AUTO_CREATE_INDEXES` instead */
  AUTO_CREATE_INDEXES: bool({ default: true }),
  MONGO_AUTO_CREATE_INDEXES: bool({ default: true }),
```